src/pages/Home.jsx

```diff
@@ -1,32 +1,32 @@
-// import { useState } from 'react';
-import { useSelector } from 'react-redux';
+import { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { getBookItems } from '../redux/books/booksSlice';
 import Card from '../components/Card';
 import AddBook from '../components/AddBook';
 
 function Home() {
-  const { bookItems, count } = useSelector((book) => book.bookItems);
-  if (count < 1) {
-    return (
-      <>
-        <div className="h-screen">
-          <p className="text-center ">Empty</p>
-        </div>
-        <AddBook />
-      </>
-    );
-  }
+  const List = useSelector((state) => state.bookItems.books);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(getBookItems());
+  }, [dispatch]);
 
   return (
     <>
-      {bookItems.map((item) => (
+      { List ? List.map((item) => (
         <Card
-          key={item.id}
-          id={item.id}
           title={item.title}
+          key={item.item_id}
+          id={item.item_id}
           author={item.author}
           category={item.category}
         />
```

Comment:
[REQUIRED] Kindly move this code to a `Booklist` component and import that component here instead. This will make your code more readable and cleaner.